### PR TITLE
[ticket/13602] Adding a new event overall_header_navbar_before

### PR DESCRIPTION
**Pull request by dragosvr92**
Event for overall_header_navbar_before.html

Event Request: http://area51.phpbb.com/phpBB/viewtopic.php?p=274786#p274786

PHPBB3-13602

### CODE REVIEW

File: phpBB/styles/prosilver/template/overall_header.html [hunk 1]
```diff
@@ -84,7 +84,7 @@
 
 			</div>
 		</div>
-
+<!-- EVENT overall_header_navbar_before -->
 		<!-- INCLUDE navbar_header.html -->
```
Comment (prototech) [inline phpBB/styles/prosilver/template/overall_header.html]: Align the EVENT with the INCLUDE.

**Comment by prototech** [comment]
Your commit messages do not follow the format specified here (https://wiki.phpbb.com/Git#Commit_Messages). Squash all commits into one. The corresponding change for subsilver2 is missing.

File: phpBB/styles/prosilver/template/overall_header.html (hunk 1 above)
Comment (dragosvr92) [inline phpBB/styles/prosilver/template/overall_header.html]: But all events are added without any spaces in front of them, or are you saying to remove the spaces of INCLUDE?

Comment (prototech) [inline phpBB/styles/prosilver/template/overall_header.html]: They're not.
[https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/styles/prosilver/template/overall_header.html#L91](overall_header_page_body_before)
[https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/styles/prosilver/template/overall_header.html#L103](overall_header_content_before)

**Comment by marc1706** [comment]
While your event description explicitly states subsilver2, you didn't add this event to subsilver2.

**Comment by dragosvr92** [comment]
Everything should be fine now, with the help of rxu.

**Comment by dhruvgoel92** [comment]
Looks good.

**Comment by RMcGirr83** [comment]
!set Event

**Comment by prototech** [comment]
The event should listed in alphabetical order in events.md. Otherwise looks ok.

**Pull request merged**